### DISCUSSION TITLE
Website: Contact Us broken

### DISCUSSION THREAD

**Discussion by leegee** [npm/feedback#621] [Website] [answered]
Submitting a support request leads to a 404 error.

**Comment by MylesBorins** [comment, marked as answer]
@leegee is this still an issue? Are you logged in when opening the support ticket? I just opened two tickets, one logged in and one logged out, and was able to get it to work.

**Comment by leegee** [comment]
It does seem to work now - thank you for checking. I came across it as part part of the 2fa sign-in when using email as the second factor authenticator.
